### PR TITLE
Add migration links

### LINKED ISSUE #376: Link migration pages in footer
Has this been linked anywhere yet? This is such a useful feature of other CDNs, is there any reason it's not already linked on the homepage under the GitHub documentation?

_Originally posted by @mattrossman in https://github.com/jsdelivr/www.jsdelivr.com/issues/270#issuecomment-747756472_

### PR DESCRIPTION
**Pull request by Drgy**
closes #376

### CODE REVIEW

**Comment by jimaek** [comment]
Maybe "Convert from" would be better? Because you dont migrate from Github, you just convert URL formats

File: src/views/components/header.html [hunk 1]
```diff
@@ -29,6 +29,12 @@
 						<ul class="dropdown-menu">
 							<li><a href="/tools/debug">Debug tool</a></li>
 							<li><a href="/tools/purge">Purge cache</a></li>
+							<li role="separator" class="divider"></li>
```
Comment (MartinKolarik) [inline src/views/components/header.html:32]: I actually meant this:
image

But your version, just without the X may be better:
image


@jimaek wdyt?

Comment (jimaek) [reply src/views/components/header.html:32]: Looks cleaner but what about SEO? Do those page have good titles, descriptions and text?

Comment (MartinKolarik) [reply src/views/components/header.html:32]: Yes, the pages themselves should be ok in that regard.

**Pull request merged**